tests/snapshots/compile_file_o_all__repeated_name_trucation.hvm.snap

```diff
@@ -3,10 +3,30 @@ source: tests/golden_tests.rs
 input_file: tests/golden_tests/compile_file_o_all/repeated_name_trucation.hvm
 ---
 Warnings:
-Mutual recursion cycle detected in compiled functions:
-  Cycle 1: long_name_that_truncates -> long_name_that_truncates$C0 -> long_name_that_truncates
-  This program will expand infinitely in strict evaluation mode.
-  Read https://github.com/HigherOrderCO/hvm-lang/blob/main/docs/lazy-definitions.md for more information.
+Seems like you're trying to run some recursive function(s) on strict-mode.
+The following recursive cycles are not compatible with strict-mode the way these functions were written:
+  * long_name_that_truncates -> long_name_that_truncates$C0 -> long_name_that_truncates
```

Comment:
Can we skip displaying generated functions? I think this will be confusing for users.